lib/rollbax/item.ex

```diff
@@ -88,6 +88,17 @@ defmodule Rollbax.Item do
     |> put_location(location)
   end
 
+  defp format_mfa_and_app(module, fun, arity) do
+    formatted = Exception.format_mfa(module, fun, arity)
+
+    case :application.get_application(module) do
```

Comment:
`Application.get_application/1` was introduced in Elixir 1.2 but we support 1.1 as well.